src/System.ObjectModel/tests/ObservableCollection/ObservableCollection_MethodsTest.netcoreapp.cs

```diff
@@ -5,6 +5,7 @@
 using System.Collections.Generic;
```

Comment:
I didn't mean to rename the files. I meant to add new files, with a partial class, and the file that is called .netcoreapp.cs contains the new tests that you're adding. The existing files remain the same. Does that make sense?

Reply:
Thanks for the clarification on this, I was not aware that was your intention at all. Please provide additional context and information in the future.

This has all been fixed in the latest code push